internal/transport/grpchttp2/http2bridge.go

```diff
@@ -0,0 +1,189 @@
+/*
+ *
+ * Copyright 2024 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package grpchttp2
+
+import (
+	"io"
+
+	"golang.org/x/net/http2"
+	"golang.org/x/net/http2/hpack"
+	"google.golang.org/grpc/mem"
+)
+
+// HTTP2FramerBridge is a struct that works as an adapter for the net/x/http2
+// Framer implementation to be able to work with the grpchttp2.Framer interface.
+// This type exists to give an opt-out feature for the new framer and it is
+// eventually going to be removed.
+type HTTP2FramerBridge struct {
+	framer *http2.Framer  // the underlying http2.Framer implementation to perform reads and writes.
+	pool   mem.BufferPool // a pool to reutilize buffers when reading.
```

Comment:
reutilize? :)
Never heard that before. But looks like it does exist.

Would `reuse` work? :)